### PR TITLE
Katello 3.18 GA

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: plugins/katello/3.18/release_notes/release_notes.md [hunk 1]
```diff
@@ -11,14 +11,22 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 * Debian content can now be managed with Pulp 3
 * System Purpose can now be set in bulk from the Content Hosts list
 * Repositories within custom products can now be associated with an OS version for better control over default enablement on Content Hosts (currently Red Hat Enterprise Linux only)
-* Content host Traces can now be resolved with hammer CLI
+* Content host Traces can now be resolved with Hammer CLI
+ * hammer
```
Comment (jeremylenz) [inline plugins/katello/3.18/release_notes/release_notes.md]: is there meant to be more here?

Comment (jturel) [reply plugins/katello/3.18/release_notes/release_notes.md]: i forgot the hammer command :(

**Review by jeremylenz — APPROVED** [review]
Changes LGTM, thanks @jturel :+1:

**Review by chris1984 — APPROVED** [review]
Release notes look good, links are working and API doc is generated. Hammer commands cover what we did for import/export. ACK

File: plugins/katello/3.18/release_notes/release_notes.md [hunk 2]
```diff
@@ -11,14 +11,22 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
...
 * System Purpose can now be set in bulk from the Content Hosts list
 * Repositories within custom products can now be associated with an OS version for better control over default enablement on Content Hosts (currently Red Hat Enterprise Linux only)
-* Content host Traces can now be resolved with hammer CLI
+* Content host Traces can now be resolved with Hammer CLI
+ * hammer host traces resolve --help
+* The ability to export content to disconnected environments has been added to support Pulp 3 repos. See the API documentation and these new Hammer commands:
+  * hammer content-export full library --help
+  * hammer content-export full version --help
+  * hammer content-export incremental library --help
+  * hammer content-export inremental version --help
```
Comment (jeremylenz) [inline plugins/katello/3.18/release_notes/release_notes.md]:
```suggestion
  * hammer content-export incremental version --help
```

**Comment by jturel** [comment]
Thanks guys!

**Pull request merged**